resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -13,7 +13,7 @@ directory:
 
     ./dev/dev-run-integration-tests.sh
 
-To run tests with Java 11 instead of Java 8, use `--java-image-tag` to specify the base image.
+To run tests with Java 21 instead of Java 17, use `--java-image-tag` to specify the base image.
 
     ./dev/dev-run-integration-tests.sh --java-image-tag 11-jre-slim
```

Comment:
Please update this according to your final change of the line 16.

Reply:
Have updated to show use of Java 17 instead of Java 21